src/Listener/ViewListener.php

```diff
@@ -12,6 +12,13 @@ class ViewListener extends BaseListener
 {
 
     /**
+     * Default associations config
+     *
+     * @var array
+     */
+    protected $associations = [];
```

Comment:
I would prefer if we follow core's standard and prefix protected properties with underscore. Other protected methods in this class are also prefixed by underscore.